x-pack/solutions/observability/plugins/synthetics/public/apps/synthetics/components/monitor_add_edit/fields/uploader.tsx

```diff
@@ -72,12 +73,9 @@ export function Uploader({ onUpload }: Props) {
   );
 }
 
-const TESTING_SCRIPT_LABEL = i18n.translate(
-  'xpack.synthetics.createPackagePolicy.stepConfigure.monitorIntegrationSettingsSection.browser.uploader.fieldLabel',
-  {
-    defaultMessage: 'Testing script',
-  }
-);
+const TESTING_SCRIPT_LABEL = i18n.translate('xpack.synthetics.monitorEdit.uploader.ariaLabel', {
+  defaultMessage: 'Input for uploading a script file for your monitor',
```

Comment:
```suggestion
  defaultMessage: 'Use this field to upload a script file for your monitor',
```

Looking at this again now I think this might be slightly nicer copy, but it's fine to ship as-is.